.prettierrc.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
   singleQuote: true,
+  trailingComma: 'es5',
+  overrides: [
```

Comment:
we only have maybe 3 js files in this repo, everything else is yaml. Would it be easier to just use double quotes everywhere and not bother with overrides?

Reply:
That was my original plan and then I thought it might be good to keep the JS consistent across all the repos.